torch/utils/hipify/__init__.py

```diff
@@ -1 +1,5 @@
-from .version import __version__
+from hipify_torch.version import __version__
```

Comment:
@lcskrishna Should we add an `import hipify_torch` here to satisfy the mypy import issue?

Reply:
Done.